docs/standards/smart-contracts/interface-ids.md

```diff
@@ -16,12 +16,12 @@ The **supportsInterface** function from **[ERC165 Standard](https://eips.ethereu
 | Contract                          | InterfaceId  | Description                                                           |
 | :-------------------------------- | :----------- | :-------------------------------------------------------------------- |
 | **ERC725X**                       | `0x44c028fe` | General executor.                                                     |
-| **ERC725Y**                       | `0x5a988c0f` | General key-value store.                                              |
+| **ERC725Y**                       | `0x714df77c` | General key-value store.                                              |
 | **LSP1UniversalReceiver**         | `0x6bb56a14` | Universal Receiver entry function.                                    |
 | **ERC1271**                       | `0x1626ba7e` | Standard Signature Validation Method for Contracts.                   |
-| **LSP0ERC725Account**             | `0x63cb749b` | Account that represent an identity on-chain                           |
+| **LSP0ERC725Account**             | `0x481e0fe8` | Account that represent an identity on-chain                           |
 | **LSP6KeyManager**                | `0x6f4df48b` | Controller for the ERC725Account.                                     |
 | **LSP1UniversalReceiverDelegate** | `0xc2d7bcc1` | Universal Receiver delegated to an other smart contract.              |
 | **LSP7DigitalAsset**              | `0xe33f65c3` | Digital Assets either fungible or non-fungible. _ERC20 A-like_        |
 | **LSP8IdentifiableDigitalAsset**  | `0x49399145` | Identifiable Digital Assets (NFT). _ERC721 A-like_                    |
-| **LSP9Vault**                     | `0x75edcee5` | Vault that could interact with other smart contracts and hold assets. |
+| **LSP9Vault**                     | `0x5e38b596` | Vault that could interact with other smart contracts and hold assets. |
```

Comment:
For which lsp-smart-contracts version this will be true?

Has this version been released yet?

Maybe we should keep history of previous `interfaceId` for older version.

These can be in the release note, changelog, etc. and we can make a link from here to it.
So developers working with previous versions of lsp-smart-contract can still easily find these `interfaceId` in case they need them.

Reply:
@Hugoo  After discussing with Fabian, he said that these interfaceIDs are not that used rn. It will be too much for adding backward compatibility for something maybe few people use.
Will change the PR name so people can browse through the history to check when and how the InterfaceID got changed.